deployment/stacks/ui-auth.ts

```diff
@@ -248,6 +249,22 @@ export function createUiAuthComponents(props: CreateUiAuthComponentsProps) {
     });
   }
 
+  new CfnOutput(scope, "CognitoIdentityPoolId", {
```

Comment:
These are needed for the CDK watch process to be able to get cognito for running locally.